_posts/2018-03-28_ncc-media-review.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: Glacier mass loss commitment
+subtitle: New Nature Climate Change study
+author: Nicolas Champollion
+date: 2018-03-28T12:00:00
+tags: ncc paper
```

Comment:
Can you use ``publication`` as tag? We try to avoid too many different tags, as they are used to classify the posts: http://oggm.org/search/

Reply:
Yes of course, I did not know which tag use ! It's indeed good to have few different tags.